docs/add-webhook-to-slack.md

```diff
@@ -2,16 +2,16 @@
 
 1. Under **"Apps & Integrations"**, search for **"Incoming Webhooks"**:
```

Comment:
Maybe provide a little more detail like, "go to your slack team's website, login, navigate to Apps & Integrations"